js/components/tabledata/index.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react';
+import {Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn} from 'material-ui/Table';
+import Diagram from '../diagram';
+
+export default class Tabledata extends Component {
+  constructor  (...args) {
+    super(...args);
+
+    this.state = {
+      multiSelectable: true,
+      enableSelectAll: false,
+      selectable: true,
+      listSelectedRows: [],
+      categoriesY: ['stargazers_count','watchers','forks','open_issues','score']
+    };
+
+    this.selectRows = this.selectRows.bind(this);
+  }
+
+  selectRows ( rows: Array<number> ) {
```

Comment:
```Array<number>``` what should do this construction?
I saw it in https://github.com/ITEAHub-bootcamp/homework-26-08-17/pull/1#discussion-diff-136442063R28